scripts/job.pbs

```diff
@@ -80,7 +71,23 @@ teardown()
 	log_echo "  $CHECKPOINT_FILE"
 	cp -p "$TEMP_DIR/checkpoint.chk" "$CHECKPOINT_FILE"
 
-	log_echo "Teardown complete, job done ;)"
+	log_echo "Teardown complete"
+}
+
+fchk()
+{
+	log_echo "Formatting checkpoint file:"
+	log_echo "  $FORMATTED_FILE"
+	
+	# creates directory for fchk files
+	mkdir -p $PBS_O_WORKDIR/fchkpoints/"
```

Comment:
This line should go at the top of the submit_jobs.bash script with the other mkdirs